### ISSUE TITLE
TRAIN AND VALIDATION LOSS SHOWING NAN 

### ISSUE THREAD

**Issue by Divyanshupy** [achaiah/pywick#10]
Hello, first of all, I would like to appreciate the work you are doing. I was trying to use the FocalLoss for Multi-Class problem and tried using FocalLoss2. It shows nan values for train and validation loss, and when I keep the gamma to 2, the training fails after one epoch while when gamma =  0(equal to cross_entropy_loss) it is working but still the loss values are nan. Please help me

**Comment by achaiah** [comment]
Hi, thanks for checking out pywick! Sorry but without knowing how you've coded your implementation I'm not sure what I can do to help. The original code for that particular Focal Loss has been moved but I think it's [here now](https://github.com/Hsuxu/Loss_ToolBox-PyTorch/blob/master/FocalLoss/focal_loss.py). You can also reach out to the original author and see if they can shed more light on their implementation. I believe it was working correctly when I first included it in the framework.

FYI there are many other losses you can use. Maybe give another one a try and see if you run into the same issues or if it gives you better results?
